doc/README_Generic_Autotools.md

```diff
@@ -0,0 +1,50 @@
+Autotools Based YaST Packages
+=============================
+
+This is a generic documenation for YaST packages which use autotools (autoconf/automake)
+for building the package.
+
+Building and Installaing
+--------------------------
+
+To build the module run these commands:
+
+    $ make -f Makefile.cvs
+    $ make
+
+If you want to rebuild the module later simply run `make` again.
+
+To install it run:
+
+    $ sudo make install
+
+Note: This will overwrite the existing module in the system, be carefull when installing
+a shared component (library) as it migth break some other modules.

```

Comment:
I'd also/rather mention that we recommend doing this in a development system instead of the production system. Just an idea.